src/components/panels/ResetPanel.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+<b-col md="6">
+  <h3 class="text-center">{{headertext}}</h3>
+  <b-list-group>
+    <b-list-group-item :style="styleObject" class="list-group-item-borderless">
+      <span class="pull-left">
+        <b>Currently it is:</b>
+      </span>
+      <TimeBadge :starttime="this.time2" :endtime="this.time1"/>
+    </b-list-group-item>
+  </b-list-group>
+</b-col>
+</template>
+
+<script>
+import TimeBadge from '@/components/TimeBadge.vue'
+import moment from 'moment'
+
+export default {
+  props: ['time', 'location'],
+  name: 'TimePanel',
+  computed: {
+    headertext() {
+      return 'Daily Reward Timer reset'
+    }
+  },
+  data() {
+    return {
+      time1: moment().toISOString(),
```

Comment:
use the time badges, not another moment calculation

Reply:
time badage need a start & end date

Reply:
and  how tell your badage the end date ?

Reply:
And why is moment needed for that? that's part of base javascript dates